### ISSUE TITLE
Special `CHECKSUM` variable only ever yields `d41d8cd98f00b204e9800998ecf8427e`

### ISSUE THREAD

**Issue by treykasada** [go-task/task#1076]
The special `CHECKSUM` variable available to `status` commands only ever yields `d41d8cd98f00b204e9800998ecf8427e`. This appears to be irrespective of the contents of any files in the `sources` list! 😮

## Environment

- Task version: `Task version: v3.22.0 (h1:sW+qi054ReRpX7nG8DcKYq3e4IzfSq366RcqEq2lDxc=)`
- Operating System: `macOS Ventura 13.2.1 (22D68)`

## Steps to reproduce

1. Create a `Taskfile.yml` file with the following contents:
   ```yaml
   version: '3'

   tasks:
     checksum:test:
       cmds:
         - cat checksum
       method: checksum
       sources:
         - '*.txt'
       status:
         - echo "{{.CHECKSUM}}" >checksum
   ```
2. Create a `file.txt` file with any contents
3. Run `task --force checksum:test` and note the printed checksum
4. Make any change to the contents of `file.txt`
5. Run `task --force checksum:test` again and note the printed checksum

## Actual results

- Both printed checksums are `d41d8cd98f00b204e9800998ecf8427e` 😢

## Expected results

- The second checksum is different from the first
- Both checksums align with the contents of `file.txt` at the time they're printed

**Comment by pd93** [comment]
@treykasada Thanks for the report. This should be fixed in #1080. As I suggested there, I don't think this ever worked. The [original PR](https://github.com/go-task/task/pull/216) tests were checking for the value that you reported is always being returned.

**Comment by treykasada** [comment]
Wow, thanks for putting up a fix so quickly @pd93! 😄
